### PR TITLE
set bins="auto" for axis histograms in pages/mutations

### LINKED ISSUE #112: Smaller bins on y-axis histogram of explorers
When looking at mutations in large datasets the time historgram is usually overwhelmed by the recent past. Narrower bins might help to get some resolution when we have horizontal banding

### PR DESCRIPTION
**Pull request by savitakartik**
added option of log-y for mutations panel
removed unused functions in plot_helpers

fixes  #112

### CODE REVIEW

**Comment by savitakartik** [comment]
https://github.com/tskit-dev/tsqc/assets/25925005/302052cd-12ad-483e-835b-39bc5fdd629c

**Comment by savitakartik** [comment]
Ok to use the same checkbox to log all Y axes here? One axis is time and the others are mutation counts.

**Comment by jeromekelleher** [comment]
Love it, great stuff

File: tsqc/pages/mutations.py [hunk 1]
```diff
@@ -32,15 +33,16 @@ def make_hist_panel(tsm, log_y):
...
 
-def page(tsm):
-    hv.extension("bokeh")
+def make_scatter_panel(log_y, tsm):
     plot_width = 1000
-    log_y_checkbox = pn.widgets.Checkbox(
-        name="Log y-axis of Mutations per site/node plots", value=False
-    )
+    y_dim = "time"
+    if log_y is True:
```
Comment (jeromekelleher) [inline tsqc/pages/mutations.py:39]: Tiny nitpick here, better to use 'if log_y' rather than comparing for identity with True

**Review by jeromekelleher — APPROVED** [review]
One nitpick, ready to merge after that

**Pull request merged**